Uplift/ViewModels/ClassesViewModel.swift

```diff
@@ -60,32 +62,35 @@ extension ClassesView {
             guard let classes = classes,
                   let selectedDate = determineDayOfMonth(selectedDay, weeksFromCurr) else { return [] }
 
-            return classes
-                .filter { toDate($0.startTime)?.isSameDay(selectedDate) == true }
-                .sorted {
-                    guard let lhsDate = toDate($0.startTime),
-                          let rhsDate = toDate($1.startTime) else { return false }
-                    return lhsDate < rhsDate
+            return classes.sorted {
+                guard let lhsDate = toDate($0.startTime),
+                      let rhsDate = toDate($1.startTime) else { return false }
+                return lhsDate < rhsDate
+            }
+            .filter {
+                if let date = toDate($0.startTime) {
+                    return date.isSameDay(selectedDate)
                 }
+                return false
+            }
         }
 
         /// The array of next sessions for this class.
         func nextSessions(classInstance: FitnessClassInstance) -> [FitnessClassInstance] {
-            guard let classes = classes,
-                  let selectedDate = toDate(classInstance.startTime) else { return [] }
-
-            return classes
-                .filter {
-                    if let date = toDate($0.startTime) {
-                        return date > selectedDate && $0.classId == classInstance.classId
-                    }
-                    return false
-                }
-                .sorted {
-                    guard let lhsDate = toDate($0.startTime),
-                          let rhsDate = toDate($1.startTime) else { return false }
-                    return lhsDate < rhsDate
+            guard let classes = classes else { return [] }
+
+            return classes.sorted {
+                guard let lhsDate = toDate($0.startTime),
+                      let rhsDate = toDate($1.startTime) else { return false }
+                return lhsDate < rhsDate
+            }
+            .filter {
+                if let date = toDate($0.startTime),
+                   let selectedDate = toDate(classInstance.startTime) {
+                    return date > selectedDate && $0.classId == classInstance.classId
                 }
+                return false
+            }
```

Comment:
Since you used this type of sorting twice, could you plz define a reusable function to do this to reduce code repetition